src/algorithms/calorimetry/CalorimeterIslandCluster.cc

```diff
@@ -130,6 +130,7 @@ void CalorimeterIslandCluster::AlgorithmInit(std::shared_ptr<spdlog::logger>& lo
           std::stringstream sstr;
           eval.print_error(sstr);
           m_log->error(sstr.str());
+          japp->Quit();
```

Comment:
Not sure how to set failure exit code here.

Reply:
@DraTeots comments on this?